packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -622,7 +622,7 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
                 }
             }
             // Sanitization: Replace multiple preceding forward slashes with just one forward slash
```

Comment:
I would update the comment here, because now it is outdated and may be confusing. 

I mean, now we are basically removing all the slashes from the path end, right?